IE7 download fails

After the language selection:
```
ERROR org.phoenicis.javafx.views.common.ErrorMessage - org.phoenicis.tools.http.DownloadException: Download of undefined has failed
	at org.phoenicis.tools.http.Downloader.get(Downloader.java:47)
	at jdk.nashorn.internal.scripts.Script$Recompilation$275$952$\^eval\_.Downloader#get([Functions, Net, Download]:46)
	at jdk.nashorn.internal.scripts.Script$Recompilation$265$1105$\^eval\_.Resource#get([Functions, Net, Resource]:68)
	at jdk.nashorn.internal.scripts.Script$219$\^eval\_.:program(<eval>:188)
	at jdk.nashorn.internal.runtime.ScriptFunctionData.invoke(ScriptFunctionData.java:637)
	at jdk.nashorn.internal.runtime.ScriptFunction.invoke(ScriptFunction.java:494)
	at jdk.nashorn.internal.runtime.ScriptRuntime.apply(ScriptRuntime.java:393)
	at jdk.nashorn.api.scripting.NashornScriptEngine.evalImpl(NashornScriptEngine.java:449)
	at jdk.nashorn.api.scripting.NashornScriptEngine.evalImpl(NashornScriptEngine.java:406)
	at jdk.nashorn.api.scripting.NashornScriptEngine.evalImpl(NashornScriptEngine.java:402)
	at jdk.nashorn.api.scripting.NashornScriptEngine.eval(NashornScriptEngine.java:155)
	at javax.script.AbstractScriptEngine.eval(AbstractScriptEngine.java:264)
	at org.phoenicis.scripts.nashorn.NashornEngine.eval(NashornEngine.java:45)
	at org.phoenicis.scripts.nashorn.NashornScriptInterpreter.runScript(NashornScriptInterpreter.java:35)
	at org.phoenicis.scripts.interpreter.BackgroundScriptInterpreter.lambda$runScript$0(BackgroundScriptInterpreter.java:35)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617)
	at java.lang.Thread.run(Thread.java:745)
Caused by: java.net.MalformedURLException: no protocol: undefined
	at java.net.URL.<init>(URL.java:593)
	at java.net.URL.<init>(URL.java:490)
	at java.net.URL.<init>(URL.java:439)
	at org.phoenicis.tools.http.Downloader.get(Downloader.java:45)
	... 17 more
```

--- a/Internet/Internet Explorer 7.0/Online/script.js
+++ b/Internet/Internet Explorer 7.0/Online/script.js
@@ -58,7 +58,7 @@ var languages = [
     'Turkish'
 ];
 var selectedLanguage = setupWizard.menu('Which language version would you like to install?', languages);
-var setupLanguage = languages[selectedLanguage.text];
+var setupLanguage = languages[selectedLanguage.index];
 
 var ie7link, ie7installer, ie7md5;
 

